src/main/kotlin/com/ampnet/identityservice/persistence/repository/UserInfoRepository.kt

```diff
@@ -6,4 +6,5 @@ import java.util.UUID
 
 interface UserInfoRepository : JpaRepository<UserInfo, UUID> {
     fun findBySessionIdOrderByCreatedAtDesc(sessionId: String): List<UserInfo>
+    fun findByFirstNameAndLastName(firstName: String, lastname: String): UserInfo
```

Comment:
This can return a list of UserInfo because `firstName` and `lastName` are not unique. Actually, this is not needed.